guide/popular-topics/reactions.md

```diff
@@ -155,6 +155,127 @@ if (message.content === '!fruits') {
 
 The benefit of this small optimization is that you can use `.then()` to handle when all of the Promises have resolved, or `.catch()` when one of them has failed. You can also `await` it since it returns a Promise itself.
 
+## Removing reactions
+
+Now that you know how to add reactions, you might be asking, how do we remove them? In this section you will learn how to remove all reactions, remove reactions by user, and remove reactions by emoji.
+
+::: warning
+All of these methods require `MANAGE_MESSAGES` permissions. Make sure your bot has permissions before attempting to utilize any of these methods, as it will error if it doesnt.
+:::
+
+### Removing all reactions
+
+Removing all reactions from a message is the easiest, the API allows us to do this through a single call. It can be done through the <branch version="11.x" inline> `message.clearReactions()` </branch> <branch version="12.x" inline> `message.reactions.removeAll()` </branch> method. 
+
+<branch version="11.x">
+
+```js
+message.clearReactions()
+	.catch(err => console.error('Failed to clear reactions: ', err));
+```
+
+</branch>
+
+<branch version="12.x">
+
+```js
+message.reactions.removeAll()
+	.catch(err => console.error('Failed to clear reactions: ', err));
+```
+
+</branch>
+
+### Removing reactions by emoji
+
+Removing reactions by emoji is not as straightforward as clearing all reactions. The API does not provide a method for selectively removing reactions by emoji, it only allows us to remove a user from a specific reaction. This means we will have to get the users who reacted with that emoji, and loop through and remove each one of them. Firstly, we need to get the reaction, for unicode emojis, this is simple as escaping the emoji in discord and copying the result as you saw before.
+
+Reaction collections are keyed by <branch version="11.x" inline>`name:id`</branch> <branch version="12.x" inline>`id`</branch> for custom emojis and by `name` for unicode emojis. This means we can simply run a `.get()` on `message.reactions` to get the reaction representing the emoji we want. After we have the reaction, we can loop through `reaction.users` and call <branch version="11.x" inline>`reaction.remove(user)`</branch> <branch version="12.x" inline>`reaction.users.remove()`</branch> on each of them.
```

Comment:
The final sentence in the paragraph before could be moved into this one, or removed entirely. It could be changed to something along the lines of:

> Reaction collections are keyed by <branch version="11.x" inline>`name:id`</branch> <branch version="12.x" inline>`id`</branch> for custom emojis and by `name` for unicode emojis (represented by their unicode character, see [here](/reactions.html#unicode-emojis)). Once you have the key, you can simply run a `.get()` on `message.reactions` to get the reaction representing the emoji you want.```